test/replication-luatest/gh_5158_invalid_recovery_from_xlog_test.lua

```diff
@@ -0,0 +1,88 @@
+local t = require('luatest')
+local cluster = require('test.luatest_helpers.cluster')
+local server = require('test.luatest_helpers.server')
+local fio = require('fio')
+
+local g = t.group('gh_5158')
+
+g.before_each(function(cg)
```

Comment:
nit: might as well be `before_all` and `after_all` instead of `after_each` below.

Reply:
If i want to use repeat test then 22 is not working properly.
I run the test like this: `.rocks/bin/luatest test/replication-luatest/xxx.lua -r 50

Reply:
Thanks for the explanation! I see now.